From 240634b1afb968a051f8c68696eae2a582a02450 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Mon, 31 Aug 2020 20:16:25 +0200
Subject: [PATCH] Rename openpgm-5.2.pc.in

This will fix the following build failure:

config.status: error: cannot find input file: `openpgm-5.3.pc.in'

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/steve-o/openpgm/pull/66]
---
 openpgm/pgm/openpgm-5.2.pc.in | 12 ------------
 openpgm/pgm/openpgm-5.3.pc.in | 12 ++++++++++++
 2 files changed, 12 insertions(+), 12 deletions(-)
 delete mode 100644 openpgm/pgm/openpgm-5.2.pc.in
 create mode 100644 openpgm/pgm/openpgm-5.3.pc.in

diff --git a/openpgm/pgm/openpgm-5.2.pc.in b/openpgm/pgm/openpgm-5.2.pc.in
deleted file mode 100644
index 9e30a6d..0000000
--- a/openpgm/pgm/openpgm-5.2.pc.in
+++ /dev/null
@@ -1,12 +0,0 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
-
-Name: OpenPGM
-Description: PGM Protocol Library.
-Version: @PACKAGE_VERSION@
-# packagers may wish to move @LIBS@ to Libs.private for platforms with
-# versions of pkg-config that support static linking.
-Libs: -L${libdir} -lpgm @LIBS@
-Cflags: -I${includedir}/pgm-@VERSION_MAJOR@.@VERSION_MINOR@
diff --git a/openpgm/pgm/openpgm-5.3.pc.in b/openpgm/pgm/openpgm-5.3.pc.in
new file mode 100644
index 0000000..9e30a6d
--- /dev/null
+++ b/openpgm/pgm/openpgm-5.3.pc.in
@@ -0,0 +1,12 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: OpenPGM
+Description: PGM Protocol Library.
+Version: @PACKAGE_VERSION@
+# packagers may wish to move @LIBS@ to Libs.private for platforms with
+# versions of pkg-config that support static linking.
+Libs: -L${libdir} -lpgm @LIBS@
+Cflags: -I${includedir}/pgm-@VERSION_MAJOR@.@VERSION_MINOR@
-- 
2.28.0

